lisa/trace.py

```diff
@@ -152,6 +152,27 @@ def add_events_deltas(self, df, col_name='delta', inplace=True):
 
         return df
 
+    def df_discretize_series(self, series, time_delta):
```

Comment:
Could we move that to datautils ? It does not really have anything to do with the trace itself since it could just work from series.index[0] to series.index[-1]. 

Also that's not really a discretization operation, but some kind of local integration (don't really know how to name that).

Reply:
> And yes, naming is hard :(

How about `series_fixed_window_integrate()` ? That's actually what it does.